portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+/** Models a calendar event. It can be used for both creation and import flow. */
+public class CalendarEvent {
+  private String name;
+  private final Instant startTimeString;
+  private final Instant endTimeString;
+
+  // TODO(hollyyuqizheng): Add an ID field if necessary.
+
+  /**
+   * Constructs a calendar event.
+   *
+   * @param name: Name for the event, can be null startTimeString: a string representation of the
```

Comment:
The comment here says name can be null, but the logic below checks for non-null.